### PR TITLE
Fix handling of unicode in torch._C._add_docstr

### PR DESCRIPTION
**Pull request by colesbury**
@apaszke, can you review the changes to THPModule_addDocStr?

### CODE REVIEW

File: torch/csrc/Module.cpp [hunk 1]
```diff
@@ -421,31 +421,49 @@ PyObject *THPModule_safeCall(PyObject *_unused, PyObject *args, PyObject *kwargs
   return result;
 }
 
+static std::string parseString(PyObject *obj)
+{
+  if (PyBytes_Check(obj)) {
```
Comment (apaszke) [inline torch/csrc/Module.cpp]: Are PyBytes even defined in Python2? I don't think so

**Review by apaszke — APPROVED** [review]
Apart from that one comment looks good to me

File: torch/csrc/Module.cpp (hunk 1 above)
Comment (apaszke) [inline torch/csrc/Module.cpp]: Weird, I can't find it in the docs

Comment (colesbury) [inline torch/csrc/Module.cpp]: https://docs.python.org/2/c-api/string.html

They're aliased to PyString in Python 2

**Pull request merged**